pygitops/operations.py

```diff
@@ -67,7 +69,9 @@ def stage_commit_push_changes(
 
     # push changes to the remote branch
     origin = repo.remotes.origin
-    push_info = origin.push(branch_name)[0]
+    if not kwargs_to_push:
+        kwargs_to_push = {}
```

Comment:
glad to see you didn't fall into the trap of trying to use `{}` as the default value 😄